server/gameObjects/Player.mjs

```diff
@@ -59,6 +61,8 @@ export default class Player {
     attack,
     attackDelayTicks,
     attackDelayMaxTicks,
+    healingDelayTicks,
+    healingDelayMaxTicks,
```

Comment:
What's the idea here, different players have different regen tick rate? Shouldn't this be global for all players?